.github/workflows/install_test.yml

```diff
@@ -0,0 +1,33 @@
+on:
+  pull_request:
+    types: [opened, synchronize]
+  push:
+    branches:
+      - main
+
+name: Install Test
+jobs:
+  install_evalml:
+    name: ${{ matrix.python_version }} install EvalML
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        python_version: ["3.7", "3.8", "3.9"]
+    steps:
+      - name: Set up python ${{ matrix.python_version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python_version }}
+      - name: Checkout repository
+        uses: actions/checkout@v2
+      - name: Install EvalML and update checker
+        run: |
+          pip config --site set global.progress_bar off
+          python -m pip install --upgrade pip
+          python -m pip install -e .[update_checker]
+      - name: Test by importing packages
+        run: |
+          python -c "import alteryx_open_src_update_checker"
+        env:
+          ALTERYX_OPEN_SRC_UPDATE_CHECKER: False
```

Comment:
@jeff-hernandez what does this env var do and why is it necessary here?

Reply:
It disables the update checker API requests, since they are not needed for testing.
- https://github.com/FeatureLabs/alteryx-open-src-update-checker#disable-checker

Reply:
Got it, thanks
Perhaps worth a code comment? I'm not sure how hard it would be for a new dev to figure that out -- they'd have to know to go look at the update checker code I guess

Reply:
I'll add a comment